src/components/CytoscapeGraph/graphs/GraphStyles.ts

```diff
@@ -626,8 +626,8 @@ export class GraphStyles {
         selector: '*.span[^isGroup]',
         style: {
           'overlay-color': PfColors.Purple200,
-          'overlay-padding': '8px',
-          'overlay-opacity': 0.5
+          'overlay-padding': '7px',
+          'overlay-opacity': 0.2
```

Comment:
@jotak, try increasing the opacity to 0.3 or maybe 0.35 and see if you are happier with the contrast.  Alternatively, or in combination, you can play with the color, try Purple300 for a slightly darker shade.

I don't see a way to get the "underlay" effect that UX was asking for, and so was looking for something that did not overly obscure the shapes.  In a separate PR I'm also lightening the graph find a little bit.